README.md

```diff
@@ -1,23 +1,8 @@
 # SignsUi
 
-To start your Phoenix server:
+To get up and running locally:
 
-  * Install dependencies with `mix deps.get`
-  * Install `yarn` with `npm install -g yarn`
-  * Install JS dependencies with `pushd assets && yarn install && popd`
-  * Start Phoenix endpoint with `NODE_ENV=development mix run --no-halt` (substituting whatever username/password you want)
-
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
-
-To get updates on the signs locally, run `realtime_signs` with a `SIGN_UI_URL` of the local url of `signs_ui`.
-See that `README` for more details.
```

Comment:
You mentioned removing this because the environment variable no longer worked — is there a new supported way to do this?